hooks/pre_start.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
-OPEN_FILES_RECOMMENDED=24576
-OPEN_FILES_LIMIT=$(ulimit -n)
+OPEN_FILES_RECOMMENDED=1024
+ULIMIT=$(ulimit -n)
+AE_OPEN_FILES_LIMIT=${AE_OPEN_FILES_LIMIT:-$ULIMIT}
+export AE_OPEN_FILES_LIMIT
 
-if [ "$OPEN_FILES_RECOMMENDED" -gt "$OPEN_FILES_LIMIT" ]; then
-    echo "WARNING: ulimit -n is $OPEN_FILES_LIMIT; $OPEN_FILES_RECOMMENDED is the recommended minimum."
+if [ "$OPEN_FILES_RECOMMENDED" -gt "$AE_OPEN_FILES_LIMIT" ]; then
+    echo "WARNING: ulimit -n is $ULIMIT; $OPEN_FILES_RECOMMENDED is the recommended minimum."
     echo "You are recommended to ensure the node is stopped and raise the maximum number of open files before starting the node."
     echo "Visit https://github.com/aeternity/aeternity/wiki/Setting-Number-of-Max-Open-Files for additional pointers."
 fi
+# prlimit doesn't automatically work on MaxOSx
```

Comment:
Could you please remove this non portable commented script

Reply:
Done